<launch>
  <arg name="robot_name" default="jaka1" />
  <arg name="robot_id" default="$(eval {'jaka1':1,'jaka2':2,'jaka3':3,'jaka4':4}.get(arg('robot_name'),1))" />
  <arg name="enable" default="true" />
  <arg name="connect_robot" default="true" />
  <arg name="calib_yaml" default="$(find jaka_close_contro)/config/world_robot_extrinsic_offline_$(arg robot_name).yaml" />
  <arg name="cube_pose_topic" default="/vision/$(arg robot_name)/cube_center_world" />
  <arg name="target_topic" default="target_tool_world" />

  <node pkg="jaka_close_contro" type="pose_servo_world_node" name="pose_servo_world" output="screen">
    <param name="robot_name" value="$(arg robot_name)" />
    <param name="robot_id" value="$(arg robot_id)" />
    <param name="enable" value="$(arg enable)" />
    <param name="connect_robot" value="$(arg connect_robot)" />
    <param name="calib_yaml" value="$(arg calib_yaml)" />
    <param name="cube_pose_topic" value="$(arg cube_pose_topic)" />
    <param name="target_topic" value="$(arg target_topic)" />
  </node>
</launch>
